aztec/src/main/kotlin/org/wordpress/aztec/AztecParser.kt

```diff
@@ -314,13 +317,6 @@ class AztecParser {
                 out.append("<li>")
             }
 
-            //special case for when cursor might be in empty list item
```

Comment:
Toggling visual and HTML with an empty list moves the cursor to the beginning of the text.  Removing this code might be the reason.  Enter the HTML below, place the cursor between the `<li>` and `</li>` tags, and toggle between views for an example.

```
Text
<ul>
  <li></li>
</ul>
```